src/main/java/stream/flarebot/flarebot/commands/moderation/PermissionsCommand.java

```diff
@@ -277,20 +288,21 @@ public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Mess
                     return;
                 } else {
                     int page = args.length == 2 ? Integer.valueOf(args[1]) : 1;
-                    Set<String> groups =
-                            this.getPermissions(channel).getGroups().stream().map(Group::getName).collect(Collectors.toSet());
-                    List<String> groupList = GeneralUtils.orderList(groups);
 
-                    String list = groupList.stream().collect(Collectors.joining("\n"));
+                    StringBuilder stringBuilder = new StringBuilder();
+                    int i = 1;
+                    for (Group group: guild.getPermissions().getGroups()) {
```

Comment:
Space before the `;` pls <3

Reply:
I would also just use a `for(int i = 1; i<= guild.getPermissions().getGroups().size(); i++)` loop as well

Reply:
Bases on this use case I think it whould be better not to